app/javascript/components/admin/AdminTable.js

```diff
@@ -423,6 +424,24 @@ class AdminTable extends React.Component {
     );
   };
 
+  /**
```

Comment:
Just a comment, but I really like the extra documentation. It's going above and beyond, and it really helps the project.